self_supervised/vision/swav.py

```diff
@@ -54,7 +54,7 @@ def get_swav_aug_pipelines(num_crops=(2,6), crop_sizes=(224,96), min_scales=(0.2
 # Cell
 class SWAV(Callback):
     order,run_valid = 9,True
-    def __init__(self, aug_pipelines=[], crop_assgn_ids=[0,1],
+    def __init__(self, aug_pipelines, crop_assgn_ids=[0,1],
```

Comment:
I would also make `crop_assgn_ids` non-optional.
The user should know which batches are the high-resolution ones.
I know this is more work, but I feel like it is easier to forget this value and make more damage, than if a _newcomer_ would have to read the documentation of the default pipeline and see which ones are the high-resolution crops.

Reply:
And I wouldn't try to infer this behind the scenes.
At least, not without documenting this behavior.
We could default it to `None` and write that we will try to infer the correct crops, but then we would have to introspect the pipeline and probably support torchvision/kornia/fastai crop transforms and retrieve their size attributes for comparison.
It is not impossible, but I think for now, we should keep it "obvious". We can always add magic later on. :)

Reply:
Done.